src/doc/python_scripting/functions.rst

```diff
@@ -3654,16 +3654,19 @@ GetLastError
 ::
 
   GetLastError() -> string
+  GetLastError(int-clr) -> string
 
 return type : string
     GetLastError returns a string containing the last error message that VisIt
-    issued.
+    issued since being cleared. If int-clr is present and is non-zero, once
+    the message is retrieved it is also cleared.
 
 
 **Description:**
 
     The GetLastError function returns a string containing the last error
-    message that VisIt issued.
+    message that VisIt issued since being cleared. If int-clr arg, if present,
+    is non-zero, once the message is retrieved, it is also cleared.
```

Comment:
this sections should match lines 3661-2.

Reply:
Whoops. Ok, fixed that one too. Also, adjusted to current policy of one sent. per line. Previously, we couldn't do that here because these strings were taken verbatim into Python doc strings. But, newest doc string generation actually does the wrapping now so I also adjusted changes paragraphs to sent. per line.